x/gamm/types/codec.go

```diff
@@ -23,6 +23,7 @@ func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&MsgExitSwapShareAmountIn{}, "osmosis/gamm/exit-swap-share-amount-in", nil)
 	cdc.RegisterConcrete(&UpdateMigrationRecordsProposal{}, "osmosis/gamm/update-migration-records-proposal", nil)
 	cdc.RegisterConcrete(&ReplaceMigrationRecordsProposal{}, "osmosis/gamm/replace-migration-records-proposal", nil)
+	cdc.RegisterConcrete(&SetScalingFactorControllerProposal{}, "osmosis/gamm/set-scaling-factor-controller-address", nil)
```

Comment:
I think this is too long to work with ledgers, cc @mattverse isn't the limit 40 characters? Or does that not apply to gov props?

Reply:
Good catch! I think it's actually 39 characters. I'm not sure if it applies to gov props

Either way, shortened it to be safe [[here](https://github.com/osmosis-labs/osmosis/pull/5937/commits/35f088d8cad643d9c3722f123aafeacbaeb14c8e)]